### DISCUSSION TITLE
As of now, opening a PWA opens in the default install of Firefox to the main tab page (not the actual PWA), instead of my default firefox-dev install (POP_OS 22.04)

### DISCUSSION THREAD

**Discussion by MostHated** [filips123/PWAsForFirefox#164] [Q&A] [answered]
Hello,
    Prior to today (or possibly a few days before, not sure, I was out of town), when I would open a PWA using ` /usr/bin/firefoxpwa site launch ABCDXYZETCETC ` would launch the PWA in firefox developers edition, which is set as my system default and I never use the default install of firefox for anything. Now, though, it always opens using the default install. What is worse is, it doesn't actually open the PWA, it just opens the default install of firefox to the main tab page, not opening the actual PWA. Is there a setting or configuration that has recently changed that I am not seeing that I need to adjust?

Thanks,
-MH

**Comment by filips123** [comment, marked as answer]
Unfortunately, there was some change in Firefox 101.0 that caused this. To fix it, you need to manually update your native program to 1.4.2.

If you added PWAsForFirefox's packagecloud repos, just run `sudo apt upgrade`. If not, download and install the correct DEB file for your architecture from [here](https://github.com/filips123/PWAsForFirefox/releases/tag/v1.4.2). The extension will say it's outdated but you can ignore it for now.

**Comment by aaronobe** [comment]
Hi @filips123 

Please I am having a similar problem to this ref #164 however I am a Windows user.

I recently had an update from FireFox to version 111.0 (64-bit), I don't know if that's caused my PWA to open up as the main tab page and not (not the actual PWA).

I have the latest version of firefoxpwa-2.4.1-x86_64.msi

Do you know why this is happening

Thank you in advance.

**Reply by filips123** [reply]
Try to enable "Always patch runtime and profile" in the PWAsForFirefox extension settings:

![settings](https://user-images.githubusercontent.com/16626308/227783857-825c35fb-7d2f-41c5-a95a-5e6d1004ec1e.png)
